src/main.rs

```diff
@@ -1,61 +1,10 @@
-mod waves;
-mod seed_generator;
-
 use std::env;
-use std::collections::HashMap;
-use std::thread;
-use std::time::Duration;
-use std::thread::JoinHandle;
-
-const ADDRESS_SIZE: usize = 35;
+use waves_address_generator::generate_seeds;
```

Comment:
Better to use relative path, isn't it?

Reply:
I'm not sure whether that is allowed in this particular case. Feel free to try it out though.